src/prefect/server/schemas/responses.py

```diff
@@ -282,3 +282,16 @@ def from_orm(cls, orm_work_queue):
             response.work_pool_name = orm_work_queue.work_pool.name
 
         return response
+
+
+class WorkPoolResponse(schemas.core.WorkPool):
+    status: Optional[schemas.statuses.WorkPoolStatus] = Field(
```

Comment:
Yeah status will be `None` for `prefect-agent` work pools. Push pools in Cloud will also have `None` for status. I'd rather not change how we communicate that in this PR because it would require some significant rework, but we can revisit this in the future if needed.

Reply:
No worries, thanks for the explanation!